contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -254,9 +292,7 @@ contract SetConfig is SetUp {
 
     bytes memory onchainConfigBytesWithBilling = abi.encode(cfg, billingTokens, billingConfigs);
 
-    uint256 a = 1234;
-    address b = ZERO_ADDRESS;
-    bytes memory offchainConfigBytes = abi.encode(a, b);
+    bytes memory offchainConfigBytes = abi.encode(1234, ZERO_ADDRESS);
```

Comment:
nitpick, but all these `offchainConfigBytes` don't really do anything yeah? Maybe we could clean up a step further just leave them blank as `""` in `setConfigTypeSafe()`

Reply:
i think for this test Lei wants to provide some coverage for `setConfig` function specifically